### PR TITLE
Android sdk v0

### PR DESCRIPTION
**Pull request by steve8708**
Screen Shot 2022-07-01 at 1 23 41 PM

### CODE REVIEW

File: packages/android/sdk/app/src/main/java/com/example/myapplication/blocks/BuilderText.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import com.example.myapplication.*
+import kotlinx.serialization.json.contentOrNull
+import kotlinx.serialization.json.jsonPrimitive
+
+@Composable
+fun BuilderText(block: BuilderBlock, text: String) {
+    val stripped = text.replace(
+        "<.*?>".toRegex(),
```
Comment (mrkoreye) [inline packages/android/sdk/app/src/main/java/com/example/myapplication/blocks/BuilderText.kt:16]: what is this doing?

File: packages/android/sdk/app/src/main/java/com/example/myapplication/ui/theme/Theme.kt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    surface = Color.White,
+    onPrimary = Color.White,
+    onSecondary = Color.Black,
+    onBackground = Color.Black,
+    onSurface = Color.Black,
+    */
+)
+
+@Composable
+fun MyApplicationTheme(
```
Comment (mrkoreye) [inline packages/android/sdk/app/src/main/java/com/example/myapplication/ui/theme/Theme.kt:31]: Why do we have this theme file if this is the SDK?

File: packages/swift/src/testing.xcodeproj/xcuserdata/stephensewell.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 3]
```diff
@@ -28,12 +28,12 @@
 		<key>testing (iOS).xcscheme_^#shared#^_</key>
 		<dict>
 			<key>orderHint</key>
-			<integer>36</integer>
+			<integer>34</integer>
```
Comment (mrkoreye) [inline packages/swift/src/testing.xcodeproj/xcuserdata/stephensewell.xcuserdatad/xcschemes/xcschememanagement.plist:31]: Why the changes in this file? It looks like maybe a local file that should not be committed?

**Review by mrkoreye — APPROVED** [review]
Why do we need all the `ic_launcher` images and xml files? Seems like there are a lot of files in here from like a project generator or something that maybe don't apply for an SDK?

File: packages/android/sdk/app/src/androidTest/java/com/example/myapplication/ExampleInstrumentedTest.kt [hunk 4]
```diff
@@ -0,0 +1,24 @@
+package com.example.myapplication
```
[the remark this answers is not in the document]
Comment (steve8708) [reply packages/android/sdk/app/src/androidTest/java/com/example/myapplication/ExampleInstrumentedTest.kt:1]: ah yes this was generated with the project, the idea is to not worry about structure yet, a next step is to rework this into a proper package which will have a name we decide (e.g. maybe like `io.builder.androidSdk`), and shuffle the files, project name, etc accordingly

File: packages/android/sdk/app/src/main/java/com/example/myapplication/MainActivity.kt [hunk 5]
```diff
@@ -0,0 +1,106 @@
+package com.example.myapplication
```
[the remark this answers is not in the document]
Comment (steve8708) [reply packages/android/sdk/app/src/main/java/com/example/myapplication/MainActivity.kt:1]: same reason as above

File: packages/android/sdk/app/src/main/java/com/example/myapplication/blocks/BuilderText.kt (hunk 1 above)
Comment (steve8708) [reply packages/android/sdk/app/src/main/java/com/example/myapplication/blocks/BuilderText.kt:16]: strips HTML, will add comment

File: packages/swift/src/testing.xcodeproj/xcuserdata/stephensewell.xcuserdatad/xcschemes/xcschememanagement.plist (hunk 3 above)
Comment (steve8708) [reply packages/swift/src/testing.xcodeproj/xcuserdata/stephensewell.xcuserdatad/xcschemes/xcschememanagement.plist:31]: no clue, I think these should probably be gitignored, will look into

File: packages/gatsby/src/batching/merge-queries.js [hunk 6]
```diff
@@ -1,345 +0,0 @@
-const { visit, visitInParallel, Kind } = require(`gatsby/graphql`);
```
Comment (mrkoreye) [inline packages/gatsby/src/batching/merge-queries.js:1]: Are these gatsby file removals supposed to be in here? Why are we removing them?

Comment (steve8708) [reply packages/gatsby/src/batching/merge-queries.js:1]: I wiped out all gitignore files. I assume they are here by mistake but @teleaziz maybe let us know if that's a wrong assumption

Comment (teleaziz) [reply packages/gatsby/src/batching/merge-queries.js:1]: those files looks needed, the gitignore is for batching folder at the root of the package which is being copied on build and publish https://github.com/BuilderIO/builder/blob/f247fa99562f5245f82cbf9dcea1143f4ebf67d3/packages/gatsby/src/constants.js#L5
